api/tests/unit/domain/services/authentication-service_test.js

```diff
@@ -93,15 +93,16 @@ describe('Unit | Domain | Services | authentication', () => {
 
   describe('#generateAccessToken', () => {
 
-    it('should return access token and id token', async () => {
+    it('should return access token, id token and expires in', async () => {
```

Comment:
`   it('should return access token, id token and validity period', async () => {`

Reply:
Ok